src/components/SingleList.jsx

```diff
@@ -2,7 +2,7 @@ import './SingleList.css';
 
 export function SingleList({ name, path, setListPath }) {
 	function handleClick() {
-		setListPath(`${name}/${path}`);
+		setListPath(path);
```

Comment:
This is from a previous issue but this is what it was supposed to be set to now that we fixed the database to the correct values.